### ISSUE TITLE
MusicSync.sh - directories not being 'cleanded'

### ISSUE THREAD

**Issue by littlejeem** [littlejeem/sync_scripts#14]
My alaclibrary_source is:

```
echo $alaclibrary_source
/home/jlivin25/Music/Library/alacimports/
```

the expectation is that the script should leave directories clean, first by deleting source files after use.

```rsync $rsync_remove_source $rsync_prune_empty $rsync_alt_vzr $alaclibrary_source $FLAC_musicdest```

should result in the source folder being empty after sync, however:

running the following after script completion

```find $alaclibrary_source -mindepth 1 -type d -print```

gives
```
/home/jlivin25/Music/Library/alacimports/Zack Hemsey
/home/jlivin25/Music/Library/alacimports/Zack Hemsey/The Way
```

**Comment by littlejeem** [comment]
this issue report is correct, there is no deletion of the top-level directories.

```$rsync_prune_empty``` equates to ```--prune-empty-dirs```

The assumption of the function of rsync of prune empty directories is in-correct, it doesn't delete empty directories AFTER the sync, it removes ALREADY empty directories from the rsync sync list so not sync't to begin with.

**Comment by littlejeem** [comment]
proposed fix 457222e doesnt work.

```
Located files in directory /home/jlivin25/Music/Downloads/
Located files in directory /home/jlivin25/Music/Library/flacimports/
Test conditions met, I would delete...
find: cannot delete ‘/home/jlivin25/Music/Downloads/Zack Hemsey/The Way’: Directory not empty
find: cannot delete ‘/home/jlivin25/Music/Downloads/Zack Hemsey’: Directory not empty
```

need a ```find``` solution that will delete through the directories
